sites/dentistryiq/server/templates/subscribe/index.marko

```diff
@@ -27,13 +27,43 @@ $ const newsletterDescription = 'Stay up-to-date on industry news and events, ne
           <endeavor-social-links block="site-subscribe" />
           <hr>
           <h5>Newsletters</h5>
-          <p>This page is currently being updated. To subscribe to Dental Economics and its newsletters, visit <a href="https://www.dentaleconomics.com/subscribe/email">Dental Economics</a>. For RDH magazine and its associated newsletter subscriptions, visit <a href="https://www.rdhmag.com/subscribe/email">RDH Magazine</a>.</p>
-          <p><strong><a href="/subscribe/email">Subscribe / Manage Preferences</a></strong></p>
+          <p>Stay up-to-date on industry news and events, new product launches and more. Select from our list of targeted newsletter offerings to stay informed on the topics that matter to you and your business.</p>
+          <h6>Dental Economics Newsletters</h6>
+          <p class="page-wrapper__description">
+            <a class="btn item__button" href="https://www.dentaleconomics.com/magazine">Subscribe to Dental Economics Newsletters</a>
+          </p>
+          <div class="page-wrapper__body">
+            <p><b>Breakthrough Clinical</b> - It’s easy to get into a clinical rut. But why stay that way? You can grow and you can change! Dr. Stacey Gividen, a general dentist—and an elite athlete—is here to help. Each month, Dr. Gividen gathers exclusive, insightful, and motivational articles that will reignite your passion for the clinical specialties. Consider this newsletter your new clinical personal trainer!</p>
```

Comment:
I know this is anal, but could you replace the `<b>` elements with `<strong>`, por favor?

Reply:
Pushed